scripts/native-system-benchmark.sh

```diff
@@ -20,26 +20,26 @@ IFS='' read -r -d '' usage <<'EOF'
 Usage: %s [options]
 
 Options:
-  -h, --help              print this help and exit
-  -c, --config=PATH       use PATH as the test configuration
-  -s, --samples=TIME      run test for TIME seconds (defaults to 30)
-                          (or indefinitely if set to inf, or infinity)
+    -h, --help              print this help and exit
+    -c, --config=PATH       use PATH as the test configuration
+    -s, --samples=TIME      run test for TIME seconds (defaults to 30)
+                            (or indefinitely if set to inf, or infinity)
 
 Cleanup:
-  --clean                 delete all previous test and preseed artifacts
-  --clean-tests           delete all previous test artifacts
-  --clean-preseeds        delete all previous preseed artifacts
+    --clean                 delete all previous test and preseed artifacts
+    --clean-tests           delete all previous test artifacts
+    --clean-preseeds        delete all previous preseed artifacts
 
 Debugging Options:
-  --debug                 run each component under a debugger
-                          (likely requires a Debug build to be useful)
-  --profile               run each component under perf
-                          (likely requires a Profiling build to be useful)
-  --leak-check            run each component under valgrind
-
-  -d, --debugger=CMD      specify the debugger CMD
-                          ("gdb" and "rr" are short-cuts for sensible defaults
-                          for those debuggers)
+    --debug                 run each component under a debugger
+                            (likely requires a Debug build to be useful)
+    --profile               run each component under perf
+                            (likely requires a Profiling build to be useful)
+    --leak-check            run each component under valgrind
+
+    -d, --debugger=CMD      specify the debugger CMD
+                            ("gdb" and "rr" are short-cuts for sensible defaults
+                            for those debuggers)
```

Comment:
This is a heredoc, a literal string that is manually-formatted to print on the terminal sensibly. Personally, I think this increase of indentation is a bit jarring (it's pretty atypical for help output to be indented so much), but I'm not particularly strong-feeling about it. However, as a matter of practice and habit, changing indentation level inside a string should absolutely not be treated the same as changing indentation level in code.

Reply:
ACK, will revert to keep this as is